.circleci/config.yml

```
@@ -58,7 +58,9 @@ workflows:
       - setup-linkcheck-build:
           filters:
             branches:
-              ignore: master,gh-pages
+              ignore:
+                - master
+                - gh-pages
```

```suggestion
              ignore: master
```
Hi @choeppler,

Thanks for the PR, but I am not sure if this will prevent the CircleCI system from running jobs on `gh-pages` branch.
Looking at the logs in CircleCI account, still it complains that no workflows found when building the `gh-pages` branch.

Alternatively, should we remove the `gh-pages` option here and then add a valid empty config file in gh-pages branch ?

---

Actually, it doesn't complain but realizes and documents that there is no workflow for the particular branch. See the tooltip if you hover over the icon before the "no workflow".

---

IMO, it now works as expected.

---

Okay, I think I got confused with the word "No workflow". In that case, this one LGTM.